src/components/StructBuilder.js

```diff
@@ -148,9 +155,10 @@ export const StructBuilder = props => {
             {
               headerRenderer: () => h(HeaderCell, ['Attribute']),
               cellRenderer: ({ rowIndex }) => {
-                const typeNamePath = buildStructTypeNamePath([rowIndex])
-                const sourcePath = buildStructSourcePath([rowIndex])
-                const sourceNamePath = buildStructSourceNamePath([rowIndex])
+                const configurationIndex = inputTableData[rowIndex].configurationIndex
```

Comment:
Could you quickly mention (maybe in a comment) what the difference between `rowIndex` and `configurationIndex` is?